src/base/bittorrent/session.cpp

```diff
@@ -1132,11 +1149,27 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
                                    .arg(ip).arg(port)
                                    , Log::INFO);
                 settingsPack.set_str(libt::settings_pack::listen_interfaces, interfacesStr);
+#ifdef Q_OS_WIN
+                chosenIP = ip;
+#endif
                 break;
             }
         }
 
+#ifdef Q_OS_WIN
+        // On Vista+ versions and after Qt 5.5 QNetworkInterface::name() returns
+        // the interface's Luid and not the GUID.
+        // Libtorrent expects GUIDs for the 'outgoing_interfaces' setting.
+        if (!networkInterface().isEmpty()) {
+            QString guid = convertIfaceNameToGuid(networkInterface());
+            if (!guid.isEmpty())
+                settingsPack.set_str(libt::settings_pack::outgoing_interfaces, guid.toStdString());
+            else
+                settingsPack.set_str(libt::settings_pack::outgoing_interfaces, chosenIP.toStdString());
```

Comment:
IMO, this substitution has to be logged to let user know about that. Sorry for the late comment.